user_events.py

```diff
@@ -756,6 +771,9 @@ def userevent_delete(id):
             __logger.info("delete user event id: %s" % sub_event_id)
             delete_user_event(sub_event_id)
 
+    if 'superEventID' in userEvent:
+        remove_subevent_from_superevent_by_eventid(id, userEvent['superEventID'])
+
```

Comment:
This is not directly related to this PR. There is the following error when deleting old sub-events:
```
File "events-manager/user_events.py", line 788, in userevent_delete
    if sub_event['id'] == platform_id:
KeyError: 'id'
```

If the `id` field in the sub-event is new, it might not be existing in old sub-events.

Changing line# 788 to the following seems to be fixing it, but please check.

```
if 'id' in sub_event and sub_event['id'] == platform_id:
```